mlir/test/Dialect/Vector/vector-transfer-to-vector-load-store.mlir

```diff
@@ -7,15 +7,13 @@ func.func @vector_transfer_ops_0d_memref(%mem: memref<f32>, %vec: vector<1x1x1xf
     %f0 = arith.constant 0.0 : f32
 
 //  CHECK-NEXT:   %[[S:.*]] = memref.load %[[MEM]][] : memref<f32>
-//  CHECK-NEXT:   %[[V:.*]] = vector.broadcast %[[S]] : f32 to vector<f32>
```

Comment:
Where is this broadcast generated? Perhaps that's something that could deleted?

Reply:
This makes sense, but what code-path generated `vector.broadcast` to begin with? And how do we know it doesn't require updating?